### ISSUE TITLE
Android crashing on launch - Unknown color

### ISSUE THREAD

**Issue by alexduckmanton** [wix/react-native-navigation#322]
### Issue Description

Hi gang, I'm having a bit of trouble getting things up and running on Android (working great on iOS). Everything seems to compile just fine, but the app crashes immediately after launching. 

### Steps to Reproduce / Code Snippets / Screenshots

This is what I'm getting from the console:

```
E/AndroidRuntime(20287): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.beetrootios/com.reactnativenavigation.controllers.NavigationActivity}: java.lang.IllegalArgumentException: Unknown color
E/AndroidRuntime(20287): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2389)
E/AndroidRuntime(20287): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2451)
E/AndroidRuntime(20287): 	at android.app.ActivityThread.access$800(ActivityThread.java:151)
E/AndroidRuntime(20287): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1354)
E/AndroidRuntime(20287): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(20287): 	at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime(20287): 	at android.app.ActivityThread.main(ActivityThread.java:5499)
E/AndroidRuntime(20287): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(20287): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime(20287): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
E/AndroidRuntime(20287): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:737)
E/AndroidRuntime(20287): Caused by: java.lang.IllegalArgumentException: Unknown color
E/AndroidRuntime(20287): 	at android.graphics.Color.parseColor(Color.java:225)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.params.StyleParams$Color.parse(StyleParams.java:34)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.params.parsers.StyleParamsParser.getColor(StyleParamsParser.java:162)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.params.parsers.StyleParamsParser.parse(StyleParamsParser.java:27)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.params.parsers.ScreenParamsParser.parse(ScreenParamsParser.java:30)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.params.parsers.ActivityParamsParser.parse(ActivityParamsParser.java:16)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.controllers.NavigationCommandsHandler.parseActivityParams(NavigationCommandsHandler.java:22)
E/AndroidRuntime(20287): 	at com.reactnativenavigation.controllers.NavigationActivity.onCreate(NavigationActivity.java:52)
E/AndroidRuntime(20287): 	at android.app.Activity.performCreate(Activity.java:5990)
E/AndroidRuntime(20287): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1117)
E/AndroidRuntime(20287): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2342)
E/AndroidRuntime(20287): 	... 10 more
```

---
### Environment
* React Native Navigation version: 2.0.0-experimental.118
* React Native version: 0.31.0
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): HTC One, Android 5.1, Debug

**Comment by guyca** [comment]
@alexduckmanton How did you define colors? Android doesn't like anything other than 6 charecter hex code like `#ffffff` etc.

**Comment by norman-kapschefsky** [comment]
@alexduckmanton I had the same problem. I debugged the color parser and a String "#F00' went in there which causes the crash. I wondered why this happen because I used this value only once for testing. Never the less, I had to **clean** the android project to make it work again.

I tested several colors on the android toolbar title but the changes didn't appeared. After cleanup they did ... hope this helps you as well
